packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js

```diff
@@ -2,13 +2,25 @@ import {
   USER_TYPE,
   PROOF_TYPE,
   TRACKING_TASK_TAG,
+  TRACKING_VARIABLES,
   LEGAL_LINK1,
   LEGAL_LINK2,
   LEGAL_LINK3,
   KYC_STATUS,
   DOCUMENTS_MATRIX,
 } from './user-identity-documents.constant';
 
+const replaceTrackingParams = (hit, params) => {
+  if (!params) return hit;
+  let formatted = hit;
+  params.forEach((param) => {
+    const { key, value } = Object.entries(param);
+    const replaced = hit.replace(`{${key}}`, value);
```

Comment:
is the final output of the replaced part supposed to be like `{to_add}_identity-files::{something}` or `{{to_add}}_identity-files::{{something}}` or `to_add_identity-files::something` because now I think it will be this case : `{to_add}_identity-files::{something}`